frontend/src/types.js

```diff
@@ -89,6 +97,9 @@ export class Clip {
     this.resolution = resolution;
     this.duplicates = duplicates;
     this.overlapping = overlapping;
+    this.frame_rate = frame_rate;
+    this.hash_sum = hash_sum;
```

Comment:
If you want you can remove hash_sum because it's only important for the backend.